content/blogs/newsletter-102.md

```diff
@@ -0,0 +1,111 @@
+While the [ReactOS Twitter account](https://twitter.com/reactos) does provide announcements, posts about working applications and such from time to time, much of what is happening with the project
+as a whole isn't mentioned. Most of what we are going to talk about is the current situation with releases and overall ReactOS development.
+
+## Why does it take so long to release a new version of ReactOS? Are you guys dead?
+The latest release of ReactOS is 0.4.14, published on 16th of December, 2021. That release alone took a year to be released. Back then the ReactOS followed the 3-months cadence, which every 3 months
+a new release would be released to the public. But since 2021, ReactOS is still at 0.4.14. Are you guys dead?
+
+The answer is certainly no, the way we handle releases has changed. Back in the day the way ReactOS releases were made it was done for the sake of quantity than quality. Every new release after 3 months
+is similar to each other, with exceptions of a very few features and bug fixes shipped to the release. But overall the net gap between the releases wasn't that big.
+Now there is a rule that for a development release to reach "Release Candidate" status, it has to have a reasonable amount of regressions (no more than 20) and that the stability isn't badly impacted
+by the introduction of new features or code changes done during the development. In other words, the ReactOS project is now focusing on providing quality releases of which the 3-months cadence cannot
+be applied anymore. Such approach wasn't quite feasible for a development team size like ours anyway.
+
+So it is expected that it will take longer for a new release to be published, although such releases will now be substantial with lots of improvements than they were before.
+If you're impatient to test the latest changes and features you can always try the [nightly builds](https://reactos.org/getbuilds) (also called experimental builds).
+
+## Development status
+In the last year and during the beginning of 2023, the ReactOS developers and contributors alike are working on many fronts of the project, the top focused area being the kernel.
+Other areas that isn't kernel related are the applications, specifically our Paint and Notepad programs, the Input Method Editor (IME) as well as other stuff such as the ReactOS
+testcase infrastructure.
+
+## ReactOS x64 port status
+The x64 port of ReactOS, while still in its infancy, has steadily gotten better in terms of stability. Back then ReactOS could barely boot and if it ever did, it usually resulted
+in an immediate crash or a BSoD by just doing simple tasks. With the help of Timo Kreuzer (tkreuzer), the x64 port has gotten to a somewhat tolerable state where ReactOS can now
+operate without big problems. Although issues still do remain, for instance audio does not currently work in x64, more apps have started to work in one way or another.
```

Comment:
That's not very accurate either. Both VMVideo and even VBoxVideo (with a small workaround) do in fact work on ReactOS x64. There was even a report of some NVidia x64 driver working. The issue is with the installer that breaks due to lack of WoW64 (at least in case of VBox). Let's just generalize and say that compatibility is currently inferior to x86 build and maybe also mention that x86 software still wouldn't run. Maybe something like this:

```suggestion
operate without big problems. Although issues still do remain and compatibility is inferior to the x86 version, more applications and even hardware drivers have started to work. Note that the x64 port currently will not run x86 applications due to lack of WoW64.
```

Reply:
@archeYR "Small workaround" means "things that are not in master" or "few hacks". AC97 is currently (and surprisingly) working under master. So... Let's say we are not compatible with x64 graphical drivers yet. The way you wrote it likes me more (less concrete). 

The same way I don't say that Intel graphic drivers works under Master x86, or the x64 version works under Real Hardware without touching anything in the code. That is simply a lie. And the people who read this must have clear the info about what can they use the x64 iso and what not.

Reply:
@julenuri
> "Small workaround" means "things that are not in master" or "few hacks". 

Not necessarily, it's just a registry change to lower the acceleration level (disable cursor acceleration). No out of tree changes are needed. If anyone is willing to install the VBox driver manually, they might as well apply the registry change in setup file which is very easy.  Apart from that, not everyone uses just VBox, VMware driver installs and works fine without any changes, and it even has OpenGL acceleration. To back my claim:
![Screenshot_20230414_204423](https://user-images.githubusercontent.com/32551254/232130737-e92c0aa0-67fa-4c15-8451-7bafd5505765.png)
> Let's say we are not compatible with x64 graphical drivers yet. 

But that's not true, some of the drivers do load and mostly work the same as if it was x86. 
> The way you wrote it likes me more (less concrete).

But it's true and gives a message to lower expectations from the x64 port in general, which is what we want in the end I guess.
> The same way I don't say that Intel graphic drivers works under Master x86, or the x64 version works under Real Hardware without touching anything in the code. That is simply a lie.

I did not say that it needs out of tree changes and I would never want to suggest anything like that in the official newsletter either...

> And the people who read this must have clear the info about what can they use the x64 iso and what not.

I think it is more important to put the correct info in the first place. I might as well say to keep the "audio doesn't work" because it indeed doesn't work in VMware on x64 (along USB and few others) and I don't care about other environments, but does it mean that audio really doesn't work at all? You seem to suggest otherwise with VBox/KVM. Also, if you've a feeling that more info is needed regarding the x64 port status and you know other specifics that apply in all cases (like missing WoW64), feel free to suggest them.

I don't think this needs to be discussed any further, but if you still want to, please move it somewhere else (ideally DM me (illen) on mattermost or matrix) to stop spamming this PR.